style.css

```diff
@@ -0,0 +1,122 @@
+html {
+  margin: 0;
+}
+
+body {
+  width: 100vw;
+  margin-top: 0;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  height: 5410px;
+  background-color: #f7f7f9;
+}
+
+button:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.mobile-main {
+  width: 100%;
+  margin: 0;
+}
+
+.mobile-header {
+  width: 100%;
+  height: 718px;
+  background: #fff;
+  border-radius: 0 0 0 90px;
+}
+
+.app-bar {
+  padding-top: 50px;
+  padding-bottom: 12px;
+  margin-left: 24px;
+  margin-right: 36px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.menu {
+  margin-top: 10px;
+}
+
+.header-main-content {
+  width: 100%;
+  height: 587.15;
+  background-image: url('/header-shapes_mobile.png');
+  background-size: 100vw 587.15px;
+  background-repeat: no-repeat;
+}
```

Comment:
Looks neat 🥇 but let's improve it by aligning the content with `flexbox`, you can do that by adding:
```
display: flex;
```
So then the content is aligned with flexbox.
 * This gives us a lot of options to align all our elements, plus, it's responsive. 😄